siriuspy/siriuspy/search/id_search.py

```diff
@@ -110,8 +110,12 @@ def conv_idname_2_idff(idname):
     def conv_idname_2_pparameter_propty(idname):
         """."""
         idff = IDSearch.conv_idname_2_idff(idname)
-        pvname = _SiriusPVName(idff['pparameter'])
-        return pvname.propty if pvname is not None else None
+        pparameter = idff['pparameter']
```

Comment:
It will always be the case that 'pparameter' is a key of the idff dictionary?

Reply:
yes, @murilobalves . for those IDs with only one type of polarization, for which pparameter is irrelevant, the valur of the key is always `None`.